RoddyCore/src/de/dkfz/roddy/config/ConfigurationConstants.java

```diff
@@ -10,6 +10,7 @@
     public static final String CFG_INPUT_BASE_DIRECTORY = "inputBaseDirectory";
     public static final String CFG_OUTPUT_BASE_DIRECTORY = "outputBaseDirectory";
     public static final String CFG_OUTPUT_ANALYSIS_BASE_DIRECTORY = "outputAnalysisBaseDirectory";
+    public static final String CFG_INPUT_ANALYSIS_BASE_DIRECTORY = "inputAnalysisBaseDirectory";
```

Comment:
`inputAnalysisBaseDirectory` was not named like this before, but exposed analogously to `outputAnalysisBaseDirectory` via `DataSet.getInputFolderForAnalysis`